etc/deploy/aws.sh

```diff
@@ -110,6 +110,7 @@ create_cloudfront_distribution() {
   sed -i 's/XXBucketNameXX/'$BUCKET'/' tmp/cloudfront-distribution.json
```

Comment:
I chose to leave this `sed` in place because there are a bunch of substitutions within that file